src/rust/generators/sysmon-generator/src/main.rs

```diff
@@ -79,7 +79,7 @@ async fn handler() -> Result<(), SysmonGeneratorError> {
 
     // TODO: also construct a stream processor for retries
 
-    let stream_processor = StreamProcessor::new_from_config(
+    let stream_processor = StreamProcessor::new(
         consumer_config,
         consumer_topic,
```

Comment:
The topics should probably go in the config object also, sorry I missed this before.

Reply:
Agreed. NBD / can be done later, but if we place the topic itno the config we could also change this `new_from_config` into a TryFrom<Config>